spec/rspec/support/caller_filter_spec.rb

```diff
@@ -43,6 +43,13 @@ def unmatched_from(files)
         expect(unmatched_from files).to eq(files)
       end
 
+      it "matches internal methods" do
+        files = %w[
+          <internal:
```

Comment:
It would be better to use real examples like the ones I posted in https://github.com/rspec/rspec-support/pull/426#issuecomment-692961012

Reply:
Yes please :)

Reply:
I've updated the PR to the real world examples you mentioned.